frontend/src/tabs/NotificationPanel.css

```diff
@@ -0,0 +1,39 @@
+.notification-panel {
```

Comment:
Can you please match this to make the styling like the other panels : 
https://github.com/keep-starknet-strange/art-peace/blob/b7122c3eedf5e2be34ea392c189e8b15d4d2f84e/frontend/src/tabs/BasicTab.css#L1